config/components/rbac/role.yaml

```diff
@@ -72,21 +72,11 @@ rules:
   - admissionregistration.k8s.io
   resources:
   - mutatingwebhookconfigurations
-  - validatingwebhookconfigurations
```

Comment:
Looking at the PR  https://github.com/kubernetes-sigs/kueue/pull/3908/files# this should not be removed.

Reply:
Yep, that was the issue for getting stuck at `waitForKueueReady`. Just pushed the changes, I think it should work now.

Reply:
This one yet: https://github.com/kubernetes-sigs/kueue/pull/3977/files#r1920444008, right?

Reply:
Ah! my bad! Sorry!